wordpress-mobile/WordPress-iOS
# Update aztec version to 1.0.0-beta 19

This PR fixes an issue where an auto completion of text was being done using smaller or larger text than the original. For example replacing the word Love for the emoji ❤️ .

To test:

- Create a new post
- Type the word Cheers
- Select the word
- Activate the emoji keyboard (tap on the globe symbol)
- see if on the autocompletion shows an emoji
- tap on the autocompletion with the emoji
- see if it not crashes.
